packages/harmony/src/storybook/colors.stories.mdx

```diff
@@ -0,0 +1,89 @@
+import { Meta, ColorPalette, ColorItem } from '@storybook/blocks'
+
+<Meta title="Colors" />
+
+<ColorPalette>
+  <ColorItem 
+    title="Static" 
+    subtitle="theme.static" 
+    colors={{
+      white: '#FFFFFF',
+      primary: '#CC0FE0'
+    }} 
+  />
+  <ColorItem 
+    title="Primary" 
+    subtitle="theme.primary" 
+    colors={{
+      primary: '#CC0FE0',
+    }}
+  />
+  <ColorItem 
+    colors={{
+      'p-100': '#D63FE6',
```

Comment:
any way we can reference these colors some way? i wonder if just a js object with the themes would be good to have in addition to our css? could maybe even use those objects for mobile in the future?

reading further looks like we actually have those objects! could we not use them here?